misc/nightly/colorer.sh

```diff
@@ -3,12 +3,12 @@
 function bcolorer {
   BIT=$1
   PLUGIN=FarColorer
-  COLORER_VERSION=$(curl -s https://raw.githubusercontent.com/colorer/FarColorer/master/version4far.txt)
+  COLORER_VERSION=$(curl -fsLJ 'https://raw.githubusercontent.com/colorer/FarColorer/master/version4far.txt')
```

Comment:
COLORER_VERSION=$(curl -fsLJ https://raw.githubusercontent.com/colorer/FarColorer/master/version4far.txt)
if [ -z "$COLORER_VERSION" ]; then
  echo "Failed to get Colorer version"
  return 1
fi